### PR TITLE
Added pull_request_template.md file

### LINKED ISSUE #3: Add pull request template
- You can refer the  docs from [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)
- Please add best practice PR template

### PR DESCRIPTION
**Pull request by Aaroh1**
Closes #3 
I've created and added a pull_request_template.md file to the .github directory.

image

### CODE REVIEW

**Comment by Aaroh1** [comment]
Hey @sohan2410! Could you please review this PR. I've added the pull request template. Let me know if something needs to be modified.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+<!-- A clear and concise description of it. -->
+
+**Additional context (OPTIONAL)**
+
+<!-- Add any other context or screenshots about the feature request here. -->
+
+**Test plan (OPTIONAL)**
+
+A good test plan should give instructions that someone else can easily follow.
```
Comment (sohan2410) [inline .github/pull_request_template.md]: Comment this line

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [x] My code follows the code style of this project.
+-->
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] The title of my pull request is a short and self descriptive.
+
+**Provide a deployed link of route/page that needs to be reviewed**
```
Comment (sohan2410) [inline .github/pull_request_template.md]: Can you please write optional here too

**Review by sohan2410 — COMMENTED** [review]
This looks great @Aaroh1! I just left a couple of comments, we can merge this once they're fixed.

**Comment by sohan2410** [comment]
Great job @Aaroh1! Now squash the commits and we are good to merge 🚀

**Comment by Aaroh1** [comment]
@sohan2410 Done!

**Pull request merged**